tests/ci/clean_up.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
 set -euo pipefail
 
+# Get OS data.
+source /etc/os-release
+
 # Restore the *-release files
 cp -fv "${TMPCI_DIR}"/os-release /etc/
 cp -fv "${TMPCI_DIR}"/redhat-release /etc/
 
-source /tmp/.env
-
 ID=${ID:-}
-IMAGE_KEY=${IMAGE_KEY:-}
+ARCH=$(arch)
+UUID=${UUID:-local}
+IMAGE_KEY="auto-${ARCH}-${UUID}"
 TEMPDIR=${TEMPDIR:-}
 HTTPD_PATH=${HTTPD_PATH:-}
 
-greenprint "🧼 Cleaning up: start"
```

Comment:
I see you added it in the commit but why are you removing greenprint from the script, what is the reason ?

Reply:
It was for removing more dependencies. I needed to declare in each one of the scripts, to source from a lib or from the common `/tmp/.env` (that how I was doing it before).
It didn't add that much value, the GitHub logs doesn't show the color.

Reply:
Ok thanks for explaining !